src/verification/verification.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { VerificationService } from './verification.service';
+
+@Controller('verification')
+export class VerificationController {
+    constructor(private readonly verificationService: VerificationService) {}
+
+    @Post('send')
+    sendVerification(@Body('userId') userId: string) {
+        const token =
+            this.verificationService.generateVerificationToken(userId);
+
+        // add logic to send the token via email or sms
+
+        return { message: 'verification sent', token };
```

Comment:
:bulb: I would probably use a [Status code Annotation](https://docs.nestjs.com/controllers#status-code) or even rely on the default 201 status code for NestJS POST requests here instead, since the consumer of our API is a Frontend which only cares about the success or failure of the `/verifification/send` request

Since this Ticket was just about the Module itself I would just add a TODO here for now:
```suggestion
        // TODO: #17 - Just send default 201 response on success, don't return data
        return { message: 'verification sent', token };
```